refactor: 🔥 remove `write_resource_batch()`, as we read from staging (#1759)
seedcase-project/seedcase-sprout: docs/design/interface/flows.qmd

    create_script --> python_script
```

## Checking data against the properties

The data must always match what is described in the properties. This means that

> the data must have the same column names, column types,> and column constraints.

Reviewer: ```suggestion
the data must have the same column names, column types, and column constraints.
```

```

::: content-hidden
<!-- TODO: Consider this more, but hide it for now. -->

## Deleting an observational unit

If you need to delete an observational unit from the data, you can use this

> flow. The `delete_observational_unit()` function will delete the observational

Reviewer 2: ```suggestion
flow. The `delete_observational_unit()` function deletes the observational
```

> unit from the data and output a list of DataFrames with the deletions.

Reviewer 2: ```suggestion
unit from the data and outputs a list of DataFrames containing the updated data.
```
